content_dev/docs/sdk/java/getting-started/keys.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: codelab
+
+title: Key Types # Step Name
+description: | # SEO Description for this step
+  Keys in Java are different from the Dart SDK AtKeys
+
+draft: false # TODO CHANGE THIS TO FALSE WHEN YOU ARE READY TO PUBLISH THE PAGE
+order: 3 # Ordering of the steps
+---
+
+### Overview
+
+If you’ve ever used our dart library before, you’ll be familiar with `AtKey`. at_java does AtKeys similarly but has few differences.
```

Comment:
Again huh? What are the differences? If there are differences please highlight them because we need to make sure there are no differences

Reply:
As far as I know, we declare AtKey objects in Dart while we declare PublicKey, SelfKey, SharedKey, PrivateHiddenKey in Java instead.

But I just removed it because I think that will confuse the reader.

2e6c0fa69d9ae034482ae4e10f4880c6e1d3c9ef

Reply:
It started off as just AtKey in the Dart library, then we introduced SelfKey, SharedKey, PublicKey etc along with factory methods. The intention is to deprecate direct use of AtKey the generic superclass, and make it abstract in a version 4+ of the client. We want to encourage use of the specific subclasses in Dart. In this respect, the Java SDK is ahead if the Dart one in that we deliberately didn't make AtKey directly usable

Reply:
Have you made changes based on my last comment?

Reply:
Yes, sorry

This is `getting-started/keys.md`
![image](https://user-images.githubusercontent.com/79019866/181837416-2b44ed25-34d9-44c6-b330-f0207187c5f6.png)

faf69f9da0aadf7d394108ace064cce50ee6357e